client/state/notices/actions.js

```diff
@@ -26,7 +25,10 @@ export function createNotice( status, text, options = {} ) {
 		isPersistent: options.isPersistent || false,
 		displayOnNextPage: options.displayOnNextPage || false,
 		status: status,
-		text: text
+		text: text,
+		button: options.button || false,
```

Comment:
I'd rather avoid these fallbacks. Usage of the property values should account for `undefined` the same as it accounts for `false` or `noop`.